kotlin/add-graphics-renderer/src/main/java/com/esri/arcgisruntime/sample/addgraphicsrenderer/MainActivity.kt

```diff
@@ -71,10 +63,39 @@ class MainActivity : AppCompatActivity() {
             renderedPointGraphicsOverlay(),
             renderedLineGraphicsOverlay(),
             renderedPolygonGraphicsOverlay(),
-          renderedCurvedPolygonGraphicsOverlay())
+            renderedCurvedPolygonGraphicsOverlay(),
+            renderedEllipseGraphicsOverlay())
         )
     }
 
+    /**
+     * Create an ellipse, its graphic, a graphics overlay for it, and add it to the map view.
+     * */
+    private fun renderedEllipseGraphicsOverlay(): GraphicsOverlay {
+        // create and set all the parameters so that the ellipse has a major axis of 400 kilometres,
+        // a minor axis of 200 kilometres and is rotated at an angle of -45 degrees.
```

Comment:
```suggestion
        // a minor axis of 200 kilometres and is rotated at an angle of -45 degrees
```

Just because the other comments in this sample aren't using periods